src/app/content/components/Page/PageContent.tsx

```diff
@@ -139,6 +139,7 @@ export default styled(MainContent)`
       &.focus {
         ${SELF_AND_CHILD_MATH_SELECTOR} {
           background-color: #ff9e4b;
+          padding: 0.2rem 0;
```

Comment:
we want this to apply to user highlights as well right? maybe we should move it up and add an `&.focus` [here](https://github.com/openstax/rex-web/pull/2101/files#diff-b3c6f238c5a4a53db657bc6f4ac089bc43d6735b106368460a1b0a2c0e2096ddR54)

Reply:
I added it to the highlight.color.focus spec.